projects/storefrontlib/src/lib/ui/layout/main/storefront.component.ts

```diff
@@ -1,8 +1,13 @@
 import { Component } from '@angular/core';
+import { SeoTitleService } from '../../../seo/seo-title.service';
 
 @Component({
   selector: 'cx-storefront',
   templateUrl: './storefront.component.html',
   styleUrls: ['./storefront.component.scss']
 })
-export class StorefrontComponent {}
+export class StorefrontComponent {
+  constructor(protected seoService: SeoTitleService) {
+    this.seoService.initPageTitle();
```

Comment:
Could we invoke the `initPageTitle` method in `APP_INITIALIZER`?

Reply:
That's a good idea.